style.css

```diff
@@ -294,7 +294,6 @@ button:hover {
 
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that your design doesn't overflow, I suggest that you do it by setting the `overflow-x` property to `hidden`, you might choose any other way out 😀.

![mobile](https://user-images.githubusercontent.com/41163571/220415797-34cc23ee-6e85-4d23-8f76-047e8c0bc163.gif)